lib/clients/graphql/__tests__/graphql_client.test.ts

```diff
@@ -223,6 +229,7 @@ describe('GraphQL client', () => {
       new ShopifyErrors.GraphqlQueryError({
         message: 'GraphQL query returned errors',
         response: errorResponse,
+        headers: expectedHeaders,
```

Comment:
If `errorResponse` contains a `headers` property, would it just be available at `response.headers`?  Why add a separate `headers` property to the `GraphqlQueryError`?